### PR TITLE
Friesland data

### LINKED ISSUE #45: Include the housing price data retrieval function into the get_data function

### LINKED ISSUE #47: Add Friesland to data retrieved
Friesland should be added to the regions Groningen and Drenthe

### PR DESCRIPTION
**Pull request by Augustin4762**
Data from Friesland municipalities added both for housing stock and housing price. Also, the housing price function added in the get_data function. Should solve the following issues: #47 and #45 .

### CODE REVIEW

File: R/get_data.R [hunk 1]
```diff
@@ -20,6 +20,13 @@ get_data <- function (data_string) {
     saveRDS(res, data_path)
     return(res)
   }
+  if (data_string == "housing_price"){
+    res <- clean_housing_price_data(
+      read_housing_price_data(),
+      read_municipality())
+    saveRDS(res, data_path)
+    return(res)
+  }
```
Comment (ikbentimkramer) [inline R/get_data.R:29]: Could you add a test for this part?

**Review by ikbentimkramer — CHANGES_REQUESTED** [review]
There is no test for the part you added to the `get_data()` function. Could you add a test for this part please?

File: R/get_data.R (hunk 1 above)
Comment (Augustin4762) [reply R/get_data.R:29]: I don't really get the mockery package, could you do it instead?

Comment (Augustin4762) [reply R/get_data.R:29]: I have tried again, can't get it done.

Comment (ikbentimkramer) [reply R/get_data.R:29]: I will write some tests for this

Comment (ikbentimkramer) [reply R/get_data.R:29]: > 
> 
> I will write some tests for this

Done

**Review by ikbentimkramer — APPROVED** [review]

**Pull request merged**